### DISCUSSION TITLE
[Feature] <New cli tool can skip the existed files when importing photos>

### DISCUSSION THREAD

**Discussion by indam** [immich-app/immich#3572] [Feature Request]
### The feature

When importing library photos to the immich database with the new CLI tool, it always traverses all the files under the import directory, even if I just added 1 new photos. This will take very long time if there are thousands of files inside.
Please add a skip option for the tool, thank you.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by etnoy** [comment]
The new v2 cli hashes files before upload
